web/src/components/wizards/common/DestinationTestPanel/DestinationTestPanel.test.tsx

```diff
@@ -0,0 +1,128 @@
+/**
+ * Panther is a Cloud-Native SIEM for the Modern Security Team.
+ * Copyright (C) 2020 Panther Labs Inc
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import {
+  render,
+  buildDestination,
+  buildSendTestAlertInput,
+  buildDeliveryResponse,
+  fireClickAndMouseEvents,
+  waitFor,
+} from 'test-utils';
+import { Wizard, WizardPanel } from 'Components/Wizard';
+import { mockSendTestAlert } from 'Source/graphql/queries';
+import DestinationTestPanel from './index';
+
+const destination = buildDestination();
+describe('DestinationTestPanel', () => {
+  it('show that destination was created successfully', () => {
+    const { getByText, container } = render(
+      <Wizard initialData={{ destination }}>
+        <Wizard.Step title="title">
+          <WizardPanel>
+            <DestinationTestPanel />
+          </WizardPanel>
+        </Wizard.Step>
+      </Wizard>
+    );
+
+    expect(getByText('Everything looks good!')).toBeInTheDocument();
+    expect(getByText('Send Test Alert')).toBeInTheDocument();
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should trigger test for alert that is successful', async () => {
+    const deliveryResponse = buildDeliveryResponse({ statusCode: 200, success: true });
+    const mocks = [
+      mockSendTestAlert({
+        variables: {
+          input: buildSendTestAlertInput({ outputIds: [destination.outputId] }),
+        },
+        data: { sendTestAlert: [deliveryResponse] },
+      }),
+    ];
+    const { getByText, container, getByAltText } = render(
+      <Wizard initialData={{ destination }}>
+        <Wizard.Step title="title">
+          <WizardPanel>
+            <DestinationTestPanel />
+          </WizardPanel>
+        </Wizard.Step>
+      </Wizard>,
+      { mocks }
+    );
+
+    expect(getByText('Everything looks good!')).toBeInTheDocument();
+    const sendTestBtn = getByText('Send Test Alert');
+    await fireClickAndMouseEvents(sendTestBtn);
+    await waitFor(() => {
+      expect(getByAltText('Test Alert received')).toBeInTheDocument();
+    });
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should trigger test for alert that is failing', async () => {
```

Comment:
can we change this? It's misleading...

```suggestion
  it('should show error details for a failing test-alert', async () => {
```

Reply:
Done!